solutions/standard/variables.tf

```diff
@@ -0,0 +1,310 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud platform API key needed to deploy IAM enabled resources."
```

Comment:
Its not only used for "IAM enabled resources" - its used for account level things too, such s2s auth policies

Reply:
updated